### ISSUE TITLE
surface HWI errors

### ISSUE THREAD

**Issue by benma** [sparrowwallet/sparrow#37]
Hi. I am a BitBox02 dev. Thanks for releasing support for the BitBox02!

While trying it out, I noticed that errors returned by HWI (e.g. by `hwi enumerate`) are not visible in the UI. In the case of the BitBox02, one error that could happen looks like this:

```
hwi enumerate
[{"error": "Could not open client or get fingerprint information: Device not paired yet. Please pair using the BitBoxApp, then close the BitBoxApp and try again.", "code": -12, "type": "bitbox02", "path": "0001:0042:00", "model": "bitbox02_multi", "needs_pin_sent": false, "needs_passphrase_sent": false}]
```

It would be helpful to show this and other errors to the user so they know what to do to fix it.

**Comment by craigraw** [comment]
Thanks for the report. It should be fixed by d8b7938.

**Comment by craigraw** [comment]
Released in 0.9.8, along with update to HWI 1.2.1
